tests/ieee802154_submac/Makefile

```diff
@@ -20,8 +20,12 @@ USEMODULE += od
 USEMODULE += shell
 USEMODULE += ps
 USEMODULE += event_thread_highest
```

Comment:
A single event_thread is used no? Can't simply `event_thread` be used?

Reply:
if it's guaranteed it run in a higher priority than the main thread, then yes

Reply:
I believe with only `event_thread` you still get the three priorities (in different queues) and with a priorit of `THREAD_PRIORITY_MAIN - 1` so should be fine and save  thread.